app/Models/NewsPost.php

```diff
@@ -77,7 +77,7 @@ public static function pageVersion()
 
     public static function search($params)
     {
-        $query = static::whereNotNull('published_at');
+        $query = static::whereNotNull('published_at')->where('published_at', '<=', Carbon::now());
```

Comment:
Can use a scope for this and `scopeDefault`.

Reply:
something like "scopePublished" and then scopedefault is that + orderby?

Reply:
yeah